samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/common.c

```diff
@@ -0,0 +1,147 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
```

Comment:
I think if you move the functions here to other files we can avoid having `common.c` and `common.h` which is something that none of  the other apps do

Reply:
If I removed common.c & common.h , then I've to restructure everything. Please allow it to be there if possible. It may be irreverent here but could help others to put  their common things in those files.

Reply:
I've make above changes except removing common.c & common.h which contains some function declarations & pre-processor definitions.

Reply:
Sounds good, we can always iterate later on future PRs.

Reply:
@carlescufi All checks have passed after latest changes. Meanwhile if possible please request @sjanc to review it to find architecture levels bugs or to scrutinize the things as per Mesh Model specification. 

Thanks for your support !!

Reply:
@carlescufi I've removed common.c as per your recommendation. Hope now you will passed it.